rhizome/host/spec/vm_setup_spec.rb

```diff
@@ -255,31 +255,31 @@ def key_wrapping_secrets
       nics = [VmSetup::Nic.new("fd48:666c:a296:ce4b:2cc6::/79", "192.168.5.50/32", "ncaka58xyg", "3e:bd:a5:96:f7:b9")]
 
       expect(vps).to receive(:write_nftables_conf).with(<<NFTABLES_CONF)
-table ip raw {
-  chain prerouting {
-    type filter hook prerouting priority raw; policy accept;
-    # allow dhcp
-    udp sport 68 udp dport 67 accept
-    udp sport 67 udp dport 68 accept
+      table ip raw {
```

Comment:
You sure you want to indent this? I know lots of people like keeping their heredocs indented with the code (using `~` usually), but I have not, because you have to get the idents just right or else you get extra whitespace characters all over the place in the literal, if I remember right. Makes it a pain to splice new input.